source/publishing_data/06_configuring_metadata/standard_metadata.rst

```diff
@@ -14,64 +14,82 @@ Refer to the table below to know more about the available metadata and how to co
   * * Themes
     * Theme(s) the dataset belongs to. Dataset themes are one of the catalog filters available in the front office of an Opendatasoft portal.
     * 1. Click on the textbox. A drop down selection of the themes defined for the domain automatically appears.
       2. Click on the chosen theme.
-      3. (optional) To add another theme to the dataset, click on the + button on the right of the textbox.
-      4. (optional) Follow the steps 1 and 2 of this procedure.
-
-      .. admonition:: Note
-         :class: note
-
-         It is also possible to add themes that are not in the drop down selection. To do so, write a new theme label in the textbox and press Enter.
+      3. (optional) To add another theme to the dataset, click repeat steps 1 and 2 of this procedure.
 
+      To delete a dataset theme, click on the cross at the right of the dataset theme name.
   * * Keywords
     * Keyword(s) to define the dataset and better find it through searches. Keywords are one of the catalog filters available in the front office of an Opendatasoft portal.
     * 1. Write a keyword in the textbox. It can contain special characters and be comprised of one or more words.
       2. (optional) A list of already used keywords appears, matching the letters just written in the textbox. It is possible to click on one of these keywords to reuse them.
       3. Once the keyword is selected or written, press Enter.
       4. (optional) To add other keywords, repeat steps 1 to 3 of this procedure.
-  * * License
-    * License attached to the dataset.
-    * 1. Click on the arrow on the right of the textbox. A drop down selection of the main open data licenses appears.
-      2. Click on the chosen license. If the right license is not in the selection, write it in that same textbox.
+
+      To delete a keyword, click on the cross at the right of that keyword.
+  * * Display modification date
+    * Date indicating when the dataset was last modified.
+    * The date can either be set manually or automatically.
+
+      * To set the date manually, click the "Update manually" button. Using the date picker displayed below the button, choose a date and time.
+      * To set the date automatically, depending on which action should lead to the modification of the date, click either or both:
+
+        * the "Update automatically when metadata has changed" button,
+        * the "Update automatically when data has changed" button.
+
+      .. admonition:: Note
+         :class: note
+
+         Depending on the dataset, some options can be unavailable.
+
+         * If the dataset is federated, the last modification date can only be set manually.
+         * If the sources are invalid, the last modification date can only be set manually, or when the metadata have changed.
```

Comment:
Shouldn't it be :
- "federated"
- "no data source" instead of "source are invalid" ?